app/src/main/java/com/android/partagix/model/Database.kt

```diff
@@ -388,6 +391,117 @@ class Database(database: FirebaseFirestore = Firebase.firestore) {
     onSuccess(user)
   }
 
+  /**
+   * Retrieve all comments that a user has received, both as a owner than as an loaner
```

Comment:
minor syntax errors:
a owner -> an owner
an loaner -> a loaner
"both as a owner than as an loaner" -> "both as an owner AND as a loaner"

Reply:
perfect